runtime-builder/Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN curl -sSL https://storage.googleapis.com/golang/go${GO_VERSION}.linux-amd64.
 
 COPY go-build.sh /usr/local/bin/
 COPY go-run.sh /usr/local/bin/
+RUN chmod 755 /usr/local/bin/go-build.sh /usr/local/bin/go-run.sh
```

Comment:
Curious.  How come we've not had this issue before?  Is it because of a difference in environment between running Travis vs doing the builds.